rtv/subscription_page.py

```diff
@@ -73,6 +106,42 @@ def close_subscriptions(self):
 
         self.active = False
 
+    @SubscriptionController.register(Command('DELETE'))
+    def delete_reddit(self):
+        "Delete the selected reddit"
+
+        context = self.content.name
+        data = self.get_selected_item()
+        listing = data['object']
+        name = data['name']
+
+        if isinstance(listing, praw.objects.Multireddit) and \
+                self.term.prompt_y_or_n('Delete {}? (y/n): '.format(name)):
+            with self.term.loader('Deleting {} from {}'.format(name, context)):
+                self.reddit.delete_multireddit(listing.name)
+                self.refresh_content()
+        else:
+            with self.term.loader('Deleting {} from {}'.format(name, context)):
```

Comment:
The language could be more specific here to avoid confusion - "Unsubscribing {} from {}"